crates/bevy_sprite/src/sprite.rs

```diff
@@ -9,7 +9,6 @@ use bevy_render::{
     texture::Texture,
 };
 use serde::{Deserialize, Serialize};
-
```

Comment:
You probably didn't mean to remove that line?

Reply:
Correct. ✅ Fixed